utility/src/main/java/org/oppia/android/util/logging/performancemetrics/ProdPerformanceMetricsUtils.kt

```diff
@@ -0,0 +1,88 @@
+package org.oppia.android.util.logging.performancemetrics
+
+import android.app.ActivityManager
+import android.content.Context
+import android.net.TrafficStats
+import org.oppia.android.app.model.OppiaMetricLog
+import org.oppia.android.util.logging.ConsoleLogger
+import java.io.File
+import javax.inject.Inject
+
+/** Utility to extract performance metrics from the underlying Android system. */
+class ProdPerformanceMetricsUtils @Inject constructor(
+  private val context: Context,
+  private val consoleLogger: ConsoleLogger
+) : PerformanceMetricsUtils {
+
+  override fun getApkSize(): Long {
+    var apkSize: Long = 0
+    try {
+      val apkPath =
+        context.packageManager.getPackageInfo(context.packageName, 0).applicationInfo.sourceDir
+      val apkFile = File(apkPath)
+      apkSize = ((apkFile.length() / 1024))
+    } catch (e: Exception) {
+      consoleLogger.e(
+        "PerformanceMetricsUtils",
+        "APK size could not be calculated."
+      )
+    }
+    return apkSize
```

Comment:
```suggestion
    return try {
      val apkPath =
        context.packageManager.getPackageInfo(context.packageName, 0).applicationInfo.sourceDir
      File(apkPath).length() / 1024
    } catch (e: Exception) {
      consoleLogger.e(
        "PerformanceMetricsUtils",
        "APK size could not be calculated."
      )
      0L
    }
```

A bit more compact and idiomatic (though it'll probably be more readable if we can eliminate the ``catch`` per my comment above).

Ditto below.

Reply:
done.